crates/extensions/tedge-p11-server/Cargo.toml

```diff
@@ -15,6 +15,7 @@ asn1-rs.workspace = true
 camino.workspace = true
 clap.workspace = true
 cryptoki.workspace = true
+percent-encoding = "2.3.1"
```

Comment:
This probably needs to be a workspace dependency (to be in-line with other deps)